README.md

```
@@ -5,195 +5,223 @@
 ![Downloads](https://img.shields.io/pypi/dm/cm-colors)
 ![License](https://img.shields.io/github/license/comfort-mode-toolkit/cm-colors)
 
-**You do your style, we make it accessible**
-_(Our color wizard can work miracles, but even magic has limits - don't expect us to make neon yellow on white look good!)_
+**Automatic color contrast tuning for web accessibility**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)